docs/pipeline.md

```diff
@@ -1,247 +1,165 @@
 # PhEval Pipeline
 
-### 1. Clone [PhEval](https://github.com/monarch-initiative/pheval)
+
+## TLDR
+
+
+The Pipeline presented on [PhEval preprint](https://www.biorxiv.org/content/10.1101/2024.06.13.598672v1) was moved to a new repository - [Monarch PhEval](https://github.com/monarch-initiative/monarch_pheval).
+
+**NOTE: The default Monarch PhEval pipeline, as proposed in the paper preprint, requires approximately 1 TB of disk space. Learn how to modify the pipeline configuration [here](#customising-pheval-pipeline-experiments) to customize the experiments.**
+
+### 1. Clone [Monarch PhEval](https://github.com/monarch-initiative/monarch_pheval)
   ```bash
-  git clone https://github.com/monarch-initiative/pheval.git
+  git clone https://github.com/monarch-initiative/monarch_pheval.git
   ```
 
-### 2. Installing PhEval dependencies
+### 2. Installing PhEval Pipeline dependencies
    Enter in the cloned folder and enter the following commands:
 
 ```bash
 poetry shell
 poetry install
 ```
 
-### 3. Generate custom Makefile
-You must have Jinja2 installed, if you don't follow the steps [here](#installing-jinja-template)
-
-In resources folder are the following files responsible for makefile generation:
+### 3. Executing Pipeline
 
-📦resources  
- ┣ 📜Makefile.j2  
- ┣ 📜custom.Makefile  
- ┣ 📜generatemakefile.sh  
- ┗ 📜pheval-config.yaml  
-
-You must edit the `pheval-config.yaml` file setting the directory where you extracted exomiser and phenotype data. An example could be found [here](#pheval-configuration-file).
-After setting the pheval-config.yaml file
-
----
-
-```mermaid
-flowchart TD
-    inputs["prepare-inputs"]
-    sr1["Setting up Runners"]
-    corpora["prepare-corpora"]
-    scrambling["Scrambing Process"]
-    r1["run"]
-    inputs ===  sr1
-    sr1 === corpora
-    corpora === scrambling
-    scrambling === r1
-```
-
----
-
-## Data Flow
-
-```mermaid
-flowchart LR
-    vcf[("Phenopackets Original Data")]
-    pheno[("Scrambled Phenopackets")]
-    result["Phenotype Result"]
-    vcf -- prepare-corpora -->  pheno
-    pheno -- scramble factor e.g 0.5 -->  result
+```bash
+make pheval
 ```
 
+## Pipeline Description
 
-## Jinja Template PhEval Makefile Generator Requirements
+The Pipeline is divided in three main steps
 
-To generate a PhEval Makefile we use the [Jinja](https://jinja.palletsprojects.com/en/3.1.x/) template engine.
+### 1. Data Preparation Phase
 
-### Installing Jinja Template
+The data preparation phase, checks the completeness of the disease, gene and variant input data and optionally preparing simulated VCF files if required, gives the user the ability to randomise phenotypic profiles using the PhEval corpus scramble command utility, allowing for the assessment of how well VGPAs handle noise and less specific phenotypic profiles when making predict.
```

Comment:
I think you should clarify that the checking of the completeness of the disease, gene & variant input data is in fact checking the data contained in the phenopacket is complete for these fields. Also, maybe this paragraph could be split just so that it flows a bit better.